### ISSUE TITLE
Allow multiple secrets under userSecretVolumeSource

### ISSUE THREAD

**Issue by MJ11FS** [instaclustr/cassandra-operator#361]
Currently if you want to use different set of certs and trust stores for internode and client connections you have to jam them under same secret under:
```
  userSecretVolumeSource:
    # name of custom secret we created with all our files
    secretName: a-secret
```

If those certs and secrets are created by an automated process then using single secret becomes tricky. Having single secret for different sets of certs etc also defies such approach. 

I recommend that `userSecretVolumeSource` is extended in a way, that it allows to specify multiple secrets and each of those is mounted as a volume.

**Comment by smiklosovic** [comment]
Hi @MJ11FS ,

thanks for the issue.

Do you think having array of secret names would do the job?

**Comment by MJ11FS** [comment]
> Do you think having array of secret names would do the job?

Hi, yes, I think it would work well.

**Comment by smiklosovic** [comment]
@MJ11FS this was released as part of version v6.0.1
